### PR TITLE
Subscriptions endpoints

### PR DESCRIPTION
**Pull request by milenakrawczyk**
Endpoints for managing subscription.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+generator client {
+  provider      = "prisma-client-js"
+  binaryTargets = ["linux-musl", "native"]
+  output        = "../generated/prisma"
+}
+
+model Subscription {
+  id                         Int       @id @default(autoincrement())
+  account                    String    @unique
```
Comment (charleslavon) [inline prisma/schema.prisma]: @milenakrawczyk would the`unique` constraint prevent `alice.near` from enabling push notifications on two clients? e.g. her desktop browser and a mobile browser

File: src/subscriptions/subscriptions.controller.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ): Promise<void> {
+    try {
+      await this.subscriptionsService.createSubscription(
+        accountId,
+        JSON.stringify(subscription),
+      );
+      console.log(`Subscription for account ${accountId} has been saved.`);
+    } catch (e: any) {
+      console.error('Subscription creation failed.', e);
+      // TODO map error
```
Comment (charleslavon) [inline src/subscriptions/subscriptions.controller.ts:31]: is console.log/error sufficient to enable logs to show up in GCP?

Comment (milenakrawczyk) [reply src/subscriptions/subscriptions.controller.ts:31]: Yes, this should be sufficient (similarly to Pagoda Console).

File: prisma/schema.prisma (hunk 1 above)
Comment (milenakrawczyk) [reply prisma/schema.prisma]: Yes, I made this simplification to show/test the full flow but might try to implement multiple subscriptions if this is preferred for the first stage.

Comment (charleslavon) [reply prisma/schema.prisma]: Yes, without multiple subscriptions per account, testing would be more difficult.

**Review by charleslavon — APPROVED** [review]

**Pull request merged**